src/openforms/js/components/admin/form_design/variables/prefill/ObjectsAPIFields.js

```diff
@@ -147,11 +156,49 @@ const ObjectsAPIFields = ({errors}) => {
           }
         >
           <ObjectTypeVersionSelect
-            name="options.objecttypeVersion"
-            apiGroupFieldName="options.objectsApiGroup"
-            objectTypeFieldName="options.objecttype"
+            name="prefillOptions.objecttypeVersion"
+            apiGroupFieldName="prefillOptions.objectsApiGroup"
+            objectTypeFieldName="prefillOptions.objecttypeUuid"
+            selectProps={{maxMenuHeight: '16em', menuPlacement: 'bottom'}}
           />
         </ErrorBoundary>
+
+        {backend ? (
+          <button
+            type="button"
+            className="button"
+            onClick={e => {
+              e.preventDefault();
+              const confirmed = window.confirm(
+                intl.formatMessage({
+                  description: `Objects API prefill configuration: warning message
+                    when copying the config from registration backend`,
+                  defaultMessage: `Copying the configuration from the registration
+                  backend will clear the existing configuration. Are you sure you want to continue?`,
+                })
+              );
+              if (confirmed) {
+                setFieldValue('prefillOptions.objectsApiGroup', backend.options.objectsApiGroup);
+                setFieldValue('prefillOptions.objecttypeUuid', backend.options.objecttype);
+                setFieldValue(
+                  'prefillOptions.objecttypeVersion',
+                  backend.options.objecttypeVersion
+                );
```

Comment:
you can also `setValues((prevValues) => ...)` to do this in a single callback and reduce the risk of race conditions

Reply:
I'll add it, where is this mapping configured for the registration backend actually? I do not see them in the objects API registration modal